server/src/main/java/org/elasticsearch/action/ingest/SimulatePipelineRequest.java

```diff
@@ -178,6 +184,10 @@ static Parsed parse(Map<String, Object> config, boolean verbose, IngestService i
                 dataMap, Fields.SOURCE);
             String index = ConfigurationUtils.readStringOrIntProperty(null, null,
                 dataMap, MetaData.INDEX.getFieldName(), "_index");
+            if (dataMap.containsKey(MetaData.TYPE.getFieldName())) {
+                deprecationLogger.deprecatedAndMaybeLog("type_in_pipeline_simulation",
+                    "Specifying _type in pipeline simulation requests is deprecated.");
+            }
             String type = ConfigurationUtils.readStringOrIntProperty(null, null,
                 dataMap, MetaData.TYPE.getFieldName(), "_type");
```

Comment:
we have been using `_doc` as a default type name elsewhere (there is a constant for it at `MapperService.SINGLE_TYPE_NAME`)

Reply:
I did notice that, but I wasn't sure if we wanted to switch it out here as this API has been using `_type` as the default for quite a while.  What are your thoughts, @jakelandis?

Reply:
+1 to changing to `_doc`